packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -264,10 +277,10 @@ class _DropdownMenuRouteLayout<T> extends SingleChildLayoutDelegate {
 
   @override
   bool shouldRelayout(_DropdownMenuRouteLayout<T> oldDelegate) {
-    return buttonRect != oldDelegate.buttonRect
-        || menuTop != oldDelegate.menuTop
-        || menuHeight != oldDelegate.menuHeight
-        || textDirection != oldDelegate.textDirection;
+    return buttonRect != oldDelegate.buttonRect ||
```

Comment:
nit: The style guide is fine with this change, but the previous way was fine too. I'd prefer to keep it aligned to the left-hand side